compiler/tests/org/jetbrains/kotlin/util/SmartIdentityTableTest.kt

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.utils
```

Comment:
Please change package to `org.jetbrains.kotlin.util`, as per the inspection that is reported here

Reply:
Ah, the test package is util and the production code package is utils. 
Fixed.